test/helpers.bash

```diff
@@ -55,7 +60,8 @@ function create_netns() {
 }
 
 function get_container_netns_path() {
-    echo /proc/$CONTAINER_NS_PID/ns/net
+    local which=$1
+    echo /proc/"${CONTAINER_NS_PIDS[$which]}"/ns/net
```

Comment:
lets make `0` the default argument so we can remove all the `get_container_netns_path 0` diffs from this PR